kotest-assertions/kotest-assertions-json/src/jvmMain/kotlin/io/kotest/assertions/json/file/matchers.kt

```diff
@@ -0,0 +1,41 @@
+package io.kotest.assertions.json.file
+
+import io.kotest.assertions.json.CompareJsonOptions
+import io.kotest.assertions.json.shouldEqualJson
+import io.kotest.assertions.json.shouldNotEqualJson
+import io.kotest.assertions.json.shouldEqualSpecifiedJson
+import io.kotest.assertions.json.shouldEqualSpecifiedJsonIgnoringOrder
+import io.kotest.assertions.json.shouldNotEqualSpecifiedJson
+import java.io.File
+
+infix fun File.shouldEqualJson(expected: String): File {
```

Comment:
I wonder if these should also be expanded to `File.shouldEqualJson(expected: file)`

This can be for another issue, but I think it fits with this PR as well

Reply:
I did originally actually intend to have all permutations of any 2 elements from the list [`File`, `Path`, `String`] for each function, however I decided against it due to
- adding many overloads
- not conforming to the existing similar functions (eg. `shouldEqualJsonFromResource`)